wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Customer.kt

```diff
@@ -0,0 +1,79 @@
+package org.stellar.walletsdk.customer
+
+import io.ktor.client.*
+import io.ktor.http.*
+import org.stellar.walletsdk.auth.AuthToken
+import org.stellar.walletsdk.exception.CustomerNotFoundException
+import org.stellar.walletsdk.exception.UnauthorizedCustomerDeletionException
+import org.stellar.walletsdk.json.toJson
+import org.stellar.walletsdk.util.Util.authDelete
+import org.stellar.walletsdk.util.Util.authGet
+import org.stellar.walletsdk.util.Util.putJson
+
+class Customer
+internal constructor(
+  private val token: AuthToken,
+  private val baseUrl: String,
+  private val httpClient: HttpClient
+) {
+  /**
+   * Get customer information by customer ID.
+   *
+   * @param id customer id
+   * @return a customer data object
+   */
+  suspend fun getById(id: String, type: String): GetCustomerResponse {
+    val endpoint = getEndpoint()
+    val urlString = "$baseUrl/$endpoint/customer?id=$id&type=$type"
+
+    return httpClient.authGet<GetCustomerResponse>(urlString, token)
+  }
+
+  /**
+   * Create a new customer.
+   *
+   * @param customerSep12Info map of customer SEP-12 information
+   * @param customerSep9Info map of customer SEP-9 information
+   * @return a customer with id information
+   */
+  suspend fun add(
+    customerSep12Info: Map<String, String>,
```

Comment:
Can you change method signature to
```
add(sep9Info: Map<String, String>, id, account, memo: Pair<String, MemoType>, type)
```
According to sep-12 spec
And check sep9 map is not empty

Reply:
I needed to update 1 rule on `detekt` so it allow me to add 6 parameter to that function. But it's done

Reply:
Nevermind this is not necessary anymore